City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-3227: High school card display

How to test:

1. Get the KASKO instance running on local and `git fetch`
2. Checkout this branch `git checkout UHF-3227_high_school_card_display`
3. Import configuration changes and clear caches`make drush-cim && make drush-cr`
4. Make sure that you have some units in your instance.
5. Go to edit or create a new landing page and name it like it would be the front page of a high school. Add some content to it, doesn't really matter what.
6. Go to edit few units in the TPR units listing. There is now a new field called "High school front page" where you can select one landing page as the front page of that unit. Select the landing page you just created and publish the unit. Publish some other units as well but don't need to add front pages for them.
7. Now go create a new page or a landing page or preferably both so you can test the feature carefully. Add high school search paragraph to the page/landing page and select the units that you published. Now the unit title should come from the front page if there is one selected and if there isn't then it is the unit title. Also if the front page is selected the unit card links to the front page - not the unit page.
8. Check that the layout works correctly on all sizes and combinations. There is quite a lot. For example if the paragraph is first after hero and there is no other content in between them the block should go "under" hero.

Layout: https://app.abstract.com/share/3fda63b5-eb44-4e2c-987b-19e0b888cc14?collectionId=3f7c640c-da80-4c62-a7a1-371980dc47db&sha=38880b057fe70204414c21e8bfc8afee24fc9e4f
